app/src/org/commcare/connect/ConnectManager.java

```diff
@@ -109,7 +115,7 @@ public interface ConnectActivityCompleteListener {
     }
 
     private static ConnectManager manager = null;
-    private ConnectIdStatus connectStatus = ConnectIdStatus.NotIntroduced;
+    private static ConnectIdStatus connectStatus = ConnectIdStatus.NotIntroduced;
```

Comment:
This shouldn't be static, can be accessed through the singleton instead, i.e. getInstance().connectStatus